src/frontend/data/genericReducers/resourceList/resourceList.spec.ts

```diff
@@ -0,0 +1,126 @@
+import { requestStatus } from '../../../types/api';
+import { modelName } from '../../../types/models';
+import { currentQuery } from './resourceList';
+
+describe('Reducer: resourceList()', () => {
+  describe('RESOURCE_LIST_GET', () => {
+    it('creates the query and marks it as pending', () => {
+      // From an empty state
+      expect(
+        currentQuery(
+          {},
+          {
+            jwt: 'some token',
+            params: {},
+            resourceName: modelName.TIMEDTEXTTRACKS,
+            type: 'RESOURCE_LIST_GET',
+          },
+        ),
+      ).toEqual({
+        currentQuery: {
+          items: {},
+          params: {
+            limit: 20,
```

Comment:
you should maybe use the default param ? `API_LIST_DEFAULT_PARAMS.limit`

Reply:
I think it's better to explicitly not use settings or variables in tests but rather hardcode stuff to make them easier to comprehend and less interconnected with the code they're testing.

WDYT?

Reply:
Ok